t/cli.c

```diff
@@ -540,10 +546,35 @@ int main(int argc, char **argv)
     }
     argc -= optind;
     argv += optind;
-    if ((ctx.certificates.count == 0) != (ctx.sign_certificate == NULL)) {
+
+    if (raw_pub_key_file != NULL) {
+        int is_dash = !strcmp(raw_pub_key_file, "-");
+        if (is_server) {
+            ctx.certificates.list = malloc(sizeof(*ctx.certificates.list));
+            load_raw_public_key(ctx.certificates.list, is_dash ? cert_location : raw_pub_key_file);
```

Comment:
Let's use `-c` / `-C` as the only option for loading the endpoint's identity.
```suggestion
            load_raw_public_key(ctx.certificates.list, cert_location);
```